src/Symfony/Bundle/FrameworkBundle/Tests/Templating/TemplateNameParserTest.php

```diff
@@ -86,4 +81,39 @@ public function testParseValidNameWithNotFoundBundle()
     {
         $this->parser->parse('BarBundle:Post:index.html.php');
     }
+
+    /**
+     * @group legacy
+     * @dataProvider provideAbsolutePaths
+     */
+    public function testAbsolutePathsAreDeprecated($name, $logicalName, $path, $ref)
+    {
+        $deprecations = array();
+        set_error_handler(function ($type, $msg) use (&$deprecations) {
+            if (E_USER_DEPRECATED === $type) {
+                $deprecations[] = $msg;
+            }
+        });
+
+        $template = $this->parser->parse($name);
+
+        restore_error_handler();
+
+        $this->assertSame($ref->getLogicalName(), $template->getLogicalName());
+        $this->assertSame($logicalName, $template->getLogicalName());
+        $this->assertSame($path, $template->getPath());
+        $this->assertCount(1, $deprecations);
+        $this->assertContains('Absolute template path support is deprecated since Symfony 3.1 and will throw an exception in 4.0.', $deprecations[0]);
+    }
+
+    public function provideAbsolutePaths()
```

Comment:
Is this a supported use case? 

Currently, and even before recent changes to the TemplateNameParser, `file:///path/to/section/index.html.php` path would be  converted to `file:path/to/section/index.html.php`